tests/integration/test_fim/test_synchronization/test_response_timeout.py

```diff
@@ -114,7 +114,7 @@ def detect_synchronization_start(time_out=1):
     def wait_agent_dbsync_finish():
         previous_time = datetime.now()
         while datetime.now() - previous_time < timedelta(seconds=3):
-            ssh_stdin, ssh_stdout, ssh_stderr = ssh.exec_command("sudo cat /var/ossec/logs/ossec.log")
+            ssh_stdin, ssh_stdout, ssh_stderr = ssh.exec_command("sudo cat /var/ossec/logs/wazuh.log")
```

Comment:
And here.

Reply:
Fixed in https://github.com/wazuh/wazuh-qa/commit/20fb78d2a4218a84aa5ce2a4323ff3437686b23b